tests/spec/sort_and_pick/test_sort_by_column_with_nulls_and_pick.py

```diff
@@ -0,0 +1,38 @@
+from ..tables import e
+
+title = "Picking the first or last row for each patient where a column contains NULLs"
+
+table_data = {
+    e: """
+          |  i1 | i2
+        --+-----+----
+        1 |     | 1
+        1 | 102 | 2
+        1 | 103 | 3
+        2 | 203 | 6
+        2 | 202 | 5
+        2 |     | 4
+        """,
+}
```

Comment:
Do we need `i2` here?

Reply:
No, I suppose we don't do we? I'm not sure now why I thought we did. I think it's worth the effort to make these specs as minimal as possible, so I'll remove.